openmc/filter.py

```diff
@@ -532,19 +554,48 @@ class CellFilter(IntegralFilter):
         filter's bins.
 
     """
+    @property
+    def bins(self):
+        return self._bins
 
+    @property
+    def num_bins(self):
+        return len(self.bins)
 
-class CellbornFilter(IntegralFilter):
-    """Bins tally events based on the cell that the particle was born in.
+    @bins.setter
+    def bins(self, bins):
+        # Format the bins as a 1D numpy array.
+        bins = np.atleast_1d(bins)
+
+        # Check the bin values.
+        cv.check_iterable_type('filter bins', bins, (Integral, openmc.Cell))
```

Comment:
Can lines 570-574 be moved into a setter at the parent `Filter` class level, leaving only lines 568 and 576-581 at the subclass level?